src/scene/pop-up.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Three.js Styled Popup Example</title>
```

Comment:
is this meant to be an example or is this the final version because I thought we were supposed to add the popup system into index.html